index.html

```diff
@@ -6,26 +6,27 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Memory Card Game</title>
   <link rel="stylesheet" type="text/css" href="styles.css">
+  <script src="scripts.js" defer="defer"></script>
```

Comment:
This is old syntax, no need for the string https://www.w3schools.com/tags/att_script_defer.asp